config/crd/bases/apps.3scale.net_apicasts.yaml

```diff
@@ -275,9 +275,34 @@ spec:
                 description: OpenSSLPeerVerificationEnabled controls OpenSSL peer
                   verification.
                 type: boolean
+              openTelemetry:
+                description: OpenTelemetry contains the gateway instrumentation configuration
+                  with APIcast.
+                properties:
+                  enabled:
+                    description: Enabled controls whether OpenTelemetry integration
+                      with APIcast is enabled. By default it is not enabled.
+                    type: boolean
+                  tracingConfigSecretKey:
+                    description: TracingConfigSecretKey if unspecified, the first
+                      key in lexicographical order will be referenced as tracing configuration.
+                      If specified, the referenced value will be used as tracing configuration.
```

Comment:
I find this a bit unclear: "the referenced value will be used as tracing configuration"?. Also, wouldn't it be better to first describe what behaviour is when specified?

Reply:
Updated